src/main/java/racing/RacingGame.java

```diff
@@ -8,21 +8,22 @@ public class RacingGame {
     public static void main(String[] args) {
 
         InputView inputView = new InputView();
-        int carCount = inputView.getCarCount();
+        String carNames = inputView.getCarNames();
+        String[] splitCarName = splitCarName(carNames);
         int retryCount = inputView.getRetryCount();
 
-        List<Car> cars = createCar(carCount);
+        List<Car> cars = createCar(splitCarName);
 
         ResultView resultView = new ResultView();
         System.out.println("실행 결과");
         raceCars(cars, retryCount, resultView);
-
+        resultView.printWinner(winner(cars));
     }
 
-    private static List<Car> createCar(int carCount) {
+    private static List<Car> createCar(String[] splitCarName) {
         List<Car> cars = new ArrayList<>();
-        for (int i = 0; i < carCount; i++) {
-            cars.add(new Car());
+        for (String s : splitCarName) {
+            cars.add(new Car(s));
```

Comment:
짧게 스쳐지나가는 변수라도 의미있게 부여해 보시면 좋을 것 같아요
```suggestion
        for (String name : splitCarName) {
            cars.add(new Car(name));
```